Authentication/Authentication/UsersNotFoundException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Authentication
+{
+    class UsersNotFoundException : FileNotFoundException
+    {
+        public UsersNotFoundException(string reason) : base(reason)
```

Comment:
Instead of inputing a reason every time, simply supply a user id which was tried to be found, but wasn't.

Reply:
This exception is thrown when the "Users.txt" file is not found, according to the Homework question -> errors, I didn't implement any system that would throw an exception if the login credentails were not found. If the login credentials are not found the program just ignores and prompt the user for choice again.